EventFilter/CTPPSRawToDigi/interface/DiamondVFATFrame.h

```diff
@@ -28,14 +31,14 @@ class DiamondVFATFrame : public VFATFrame
     /// get timing infromation
     uint32_t getLeadingEdgeTime() const
     {
-      uint32_t time = ((data[5]&0x1f)<<16)+data[6];
+      uint32_t time = ((data[7]&0x1f)<<16)+data[8];
```

Comment:
just curious: is this a data format change or a bugfix (that the leading and trailing were actually swapped)